pkg/apis/serving/v1alpha1/service_defaults_test.go

```diff
@@ -121,7 +117,7 @@ func TestServiceDefaulting(t *testing.T) {
 					Configuration: ConfigurationSpec{
 						RevisionTemplate: RevisionTemplateSpec{
 							Spec: RevisionSpec{
-								ConcurrencyModel: "Single",
```

Comment:
Can you add a test which only specifies `ConcurrencyModel: "Single"` and make sure that the Service is accepted?

Reply:
Tested at Revision and KPA level.